packages/nyc.config.js

```diff
@@ -1,3 +1,5 @@
+const defaults = require("@istanbuljs/schema").defaults
```

Comment:
Well this is definitively useful. 

I had issues with nyc in the past with nyc excludes/includes because I did not know where to get the defaults :)

Reply:
haha, yeah so did I.  then I busted out my google-fu 😤